net.supertuxkart.SuperTuxKart.yml

```diff
@@ -0,0 +1,46 @@
+app-id: net.supertuxkart.SuperTuxKart
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+rename-desktop-file: supertuxkart.desktop
+rename-appdata-file: supertuxkart.appdata.xml
```

Comment:
Remove this. The AppStream file is not being renamed anymore. Only the desktop file is.